### ISSUE TITLE
Add Address.fromString conversion

### ISSUE THREAD

**Issue by Jannis** [graphprotocol/graph-cli#86]
There are situations where developers will want to hard-code contract addresses in mappings, in order to bind them with e.g.

```typescript
let contract = SomeContract.bind(Address.fromString("0x..."), event.blockHash)
...
```

However, there no way to convert from a string to an address in our mappings.

**Comment by Jannis** [comment]
Nope, overloading results in TypeScript compile errors like this one:
```
ERROR TS2300: Duplicate identifier 'index/LANDRegistry.bind'.

   static bind(address: string, blockHash: H256): LANDRegistry {
          ~~~~
 in index.ts(919,9)
```

**Comment by Jannis** [comment]
We _may_ be able to use overload lists for this, assuming AssemblyScript allows it: https://blog.mariusschulz.com/2016/08/18/function-overloads-in-typescript#version-4-function-overloads. I can try that.

**Comment by Jannis** [comment]
Nah, doesn't work with AssemblyScript.

**Comment by MaxGraey** [comment]
It works in AssemblyScript but with some limitations. For now you need explicitly resolve all generic types for overloaded functions. In future this will be more fancy.

See [this example](https://webassembly.studio/?f=04uwp09hkygd)

**Comment by Jannis** [comment]
@MaxGraey Oh sweet, we'll give that a shot. It looks good enough for the moment.

**Comment by leodasvacas** [comment]
What is this tracking now? Having the overloaded version? Because we do have `Address.fromString`.

**Comment by Jannis** [comment]
Yeah, multiple implementations of `SmartContract.bind()` (one with an `Address`, one with a `string` parameter). It is a low priority issue right now.

**Comment by incrypto32** [comment]
@schmidsi since Address.fromString() is now available. i guess we can close this?
